pages/reference/base-images/balena-base-images.md

```diff
@@ -1,171 +1,197 @@
-- `<distro>` is the Linux distribution. Currently there are 4 distributions, namely Debian, Alpine, Ubuntu and Fedora. This field is optional and will default to Debian if left out.
-- `<lang_stack>` is the programming language pack, currently we support Node.js, Python, OpenJDK, .Net, and Go. This field is optional, and if left out, no language pack will be installed, so you will just have the distribution and you can later install and use any language in your image/container.
+Dockerhub uses a different naming convention for device architecture than balena. Use the table below to convert a balena specified architecture to the equivalent architecture listed on Dockerhub. To see the architecture for balena-supported devices, check out the [balena Machine names and architecture page](https://docs.balena.io/reference/base-images/devicetypes/).
 
-#### Image Tags
+As an example, if you are using a Raspberry Pi Zero 2W, the balena machine name page shows the “balena_arch” as “aarch64”. In the table below, aarch64 is referred to as  “arm64v8 (ARMv8 64-bit)” on Dockerhub. Look out for arm64v8 images to find images compatible with the Raspberry Pi Zero 2W device type. 
 
-In the tags, all of the fields are optional, and if they are left out, they will default to their `latest` pointer.
 
-- `<lang_ver>` is the version of the language stack, for example, Node.js 20.12.0, it can also be substituted for `latest`.
-- `<distro_ver>` is the version of the Linux distro, for example Debian and it's valid version as mentioned above in the list.
-- For each combination of distro and stack, we have two variants called `run` and `build`. The build variant is much heavier as it has a number of tools preinstalled to help with building source code. You can see an example of the tools that are included in the [Debian variants][debian-variants]. Navigate to the Distro version you are looking for and find the `build` and `run` variants of the image. The `run` variants are stripped down and only include a few useful runtime tools. If no variant is specified, the image defaults to `run`.
-- The last optional field on tags is the date tag `<yyyymmdd>`. If a date tag is specified, the pinned release will always be pulled from Docker Hub, even if there is a new one available. 
+| Docker Architecture | Balena Architecture (BALENA\_ARCH) | Alternative Names | Example Device Types |
+| :---- | :---- | :---- | :---- |
+| arm32v6 (ARMv6 32-bit) | rpi | armv6, armhf (in some contexts) | Raspberry Pi 1 model B+, Raspberry Pi Zero/W |
+| arm32v7 (ARMv7 32-bit) | armv7hf | armv7, armv7l, armhf | Raspberry Pi 2, Raspberry Pi 3 model B+ |
+| arm64v8 (ARMv8 64-bit) | aarch64 | arm64, armv8 | Raspberry Pi 3 64-bit, Raspberry Pi 4, NVIDIA Jetson |
```

Comment:
Also Raspberry Pi 5 and Raspberry Pi Zero 2 W

Reply:
The list is meant to be for reference purposes only. 
@alanb128 I might further recommend not using all examples as RPi and add from different manufacturers too if available.

Reply:
That's fair :+1: